readalongs/align.py

```diff
@@ -236,30 +236,30 @@ def create_asr_config(
 
     Returns:
         soundswallower.Config: Basic configuration."""
-    asr_config = soundswallower.Decoder.default_config()
+    asr_config = soundswallower.Config()
```

Comment:
I like this interface, it's much nicer.

Reply:
Yes - in JavaScript actually there is no more Config class, because configuration and initialization of the decoder are done separately.  It isn't ideal that the APIs have diverged in this way but they diverge in other ways too, largely due to the asynchronous nature of the JS code.